examples/persist-create/persist_create.bal

```diff
@@ -0,0 +1,44 @@
+import ballerina/io;
+import rainier.store;
```

Comment:
can we order the imports properly.

Reply:
Fixed in [c1be323](https://github.com/ballerina-platform/ballerina-distribution/pull/4429/commits/c1be3230aa93ba7a9aaa73823a67994b0d1e85fe)